src/sfloader/fluid_defsfont.c

```diff
@@ -832,43 +832,40 @@ fluid_defpreset_noteon(fluid_defpreset_t *defpreset, fluid_synth_t *synth, int c
                     for(i = 0; i < GEN_LAST; i++)
                     {
 
-                        /* SF 2.01 section 8.5 page 58: If some generators are
-                         * encountered at preset level, they should be ignored */
-                        if((i != GEN_STARTADDROFS)
-                                && (i != GEN_ENDADDROFS)
-                                && (i != GEN_STARTLOOPADDROFS)
-                                && (i != GEN_ENDLOOPADDROFS)
-                                && (i != GEN_STARTADDRCOARSEOFS)
-                                && (i != GEN_ENDADDRCOARSEOFS)
-                                && (i != GEN_STARTLOOPADDRCOARSEOFS)
-                                && (i != GEN_KEYNUM)
-                                && (i != GEN_VELOCITY)
-                                && (i != GEN_ENDLOOPADDRCOARSEOFS)
-                                && (i != GEN_SAMPLEMODE)
-                                && (i != GEN_EXCLUSIVECLASS)
-                                && (i != GEN_OVERRIDEROOTKEY))
+ 						/* SF 2.01 section 8.5 page 58: If some generators are
```

Comment:
Can you format this comment properly and make sure the indentation is correct?

Reply:
Yes.